README.md

```diff
@@ -117,21 +147,29 @@ with a ***clear description*** and **_illustrative_ screenshots**:
 
 [![create-issue-for-home-screen-icon](https://cloud.githubusercontent.com/assets/24758860/22227322/ae12d9f6-e1c2-11e6-928c-c20cfe8b3912.png)](https://github.com/dwyl/dwyl.github.io/issues/194)
 
-This is a _real_ issue: https://github.com/dwyl/dwyl.github.io/issues/194
-if you want to see how it was _resolved_, click and go check it out!
+> This is a _real_ issue: https://github.com/dwyl/dwyl.github.io/issues/194
+**you _can_ help us _resolve_ it**!
+
+
 
 ### C) Submit the _New Issue_!
 
-+ Once you _submit_ a new issue for a User Story or Issue/Bug a member
+1. Preview the issue _before_ you submit it to ensure any images were uploaded
```

Comment:
Is this really necessary? When you hit 'submit' you're taken to the submitted issue and can see straight away if something is amiss.

I would suggest removing this for brevity.